static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner-parameters.js

```diff
@@ -58,24 +84,8 @@ const supporterCost = (
             : euro + amount;
     }
 
-    const payment = {
-        MONTHLY: {
-            GB: '£5',
-            US: '$6.99',
-            AU: '$10',
-            CA: '$6.99',
-            INT: '$6.99',
-        },
-        'ONE-OFF': {
-            GB: '£1',
-            US: '$1',
-            AU: '$1',
-            CA: '$1',
-            INT: '$1',
-        },
-    }[contributionType][region];
-
-    return payment || '£5';
+    return `${extendedCurrencySymbol[region]}
+            ${paymentAmounts[contributionType][region]}`;
 };
 
 const supporterEngagementCtaCopy = (location: string): string =>
```

Comment:
OK, not in your changes but I think this function could be simplified to make things less confusing. It has a special case for US - but it will never be called in the US because the US gets `supporterParams` which gets `supporterEngagementCtaCopyJustOne`. So remove the special case. And also rename these functions to `recurringCtaCopy` and `oneOffCtaCopy`. What do you think?

Reply:
Fixed in 487079f